docs/application/dotnet/guides/nui/window.md

```diff
@@ -109,57 +123,76 @@ If you touch the window, the touch event is triggered. If multi points are touch
 
 The following code shows how to handle the touch event:
 
-```csharp
-window.TouchEvent += OnWindowTouched;
+1. Create a `TouchEvent` handler:
 
-private void OnWindowTouched(object sender, Window.TouchEventArgs e)
-{
-    if (e.Touch.GetState(0) == PointStateType.Down)
+    ```csharp
+    private void OnWindowTouched(object sender, Window.TouchEventArgs e)
     {
-        // The window has been touched, do something
+        if (e.Touch.GetState(0) == PointStateType.Down)
+        {
+            // The window has been touched, do something
+        }
     }
-}
-```
+    ```
+2. Add above handler to window's the `TouchEvent` property:
+
+    ```csharp
+    window.TouchEvent += OnWindowTouched;
+    ```
 
 ### Key event
 
 Key event is emitted when the window receives a key event from the window manager.
 
 The following code shows how to handle the key event:
 
-```csharp
-window.KeyEvent += OnWindowKeyEvent;
+1. Create a `KeyEvent` handler:
 
-private void OnWindowKeyEvent(object sender, Window.KeyEventArgs e)
-{
-    if (e.Key.State == Key.StateType.Down)
+    ```csharp
+    private void OnWindowKeyEvent(object sender, Window.KeyEventArgs e)
     {
-        if (e.Key.KeyPressedName == "Left")
+        if (e.Key.State == Key.StateType.Down)
         {
-            // The left arrow key is pressed, do something
-        }
-        else if (e.Key.KeyPressedName == "Right")
-        {
-            // The right arrow key is pressed, do something
+            if (e.Key.KeyPressedName == "Left")
+            {
+                // The left arrow key is pressed, do something
+            }
+            else if (e.Key.KeyPressedName == "Right")
+            {
+                // The right arrow key is pressed, do something
+            }
         }
     }
-}
-```
+    ```
+
+2. Add above handler to the window's `KeyEvent` property:
+
+    ```csharp
+    window.KeyEvent += OnWindowKeyEvent;
+    ```
 
 ### Resize event
 
 Resize event is emitted when the window is resized.
 
 The following code shows how to handle the resize event:
 
-```csharp
-window.Resized += OnWindowResized;
+1. Create a `Resized` handler:
```

Comment:
Since property is called `Resized`:
`window.Resized += OnWindowResized;`
I decided to use this name, but you are right. Maybe we should use `ResizeEvent` instead?

Reply:
@pkalota yes that will be better.

Reply:
@mijin85cho thanks.